### PR TITLE
Added method for exporting DataSet as PKML

### LINKED ISSUE #578: DataSet: save as pkml
Save the DataRepositry as PKML. Would implement it as a method of `DataSet`

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #578

### CODE REVIEW

File: R/data-set.R [hunk 1]
```diff
@@ -197,6 +197,18 @@ DataSet <- R6::R6Class(
       }
     },
 
+    #' @description
+    #' Save the \code{DataSet} to a pkml file that can be loaded by MoBi
+    #' @param filePath Path to the location (incl. file name)
+    saveToPKML = function(filePath){
```
Comment (msevestre) [inline R/data-set.R:203]: In order to stay consistent with what we have done for other objects, shouldn't we create a utility-data-set or sthg and define this method there instead? Also this is were you could create some help methods around dataSet such as createDataSet(x=, y= etc..)

Comment (PavelBal) [reply R/data-set.R:203]: Ok, then "saveDataSetToPKML" or "export..."? We are using "export" for simulation results etc, but "save" sounds somehow more intuitive for me..

Comment (msevestre) [reply R/data-set.R:203]: saveDataSetToPKML
We can also create aliases for other method (so that we are consistent and always have save..xx)

Comment (PavelBal) [reply R/data-set.R:215]: @msevestre We would have to expose the `DataRepository` of the `DataSet` in order to save it via a utility function...

Comment (msevestre) [reply R/data-set.R:215]: Hontestly I do not have an issue with exposing the underlying DataREpository as public readonly {get;}. We can create the object tusign a DataRepo as parameter so why not?

Comment (PavelBal) [reply R/data-set.R:215]: ok, I will expose it (readonly).

Comment (msevestre) [reply R/data-set.R:215]: any update on this? Id' like to merge asap

**Comment by PavelBal** [comment]
@msevestre Updated.

**Pull request merged**